tunnel/proxy.go

```diff
@@ -27,149 +23,86 @@ func endpointHost(region string) string {
 // Dialer is a proxy destination dialer.
 type Dialer func() (io.ReadWriteCloser, error)
 
-func (t *tunnel) proxy(ctx context.Context, dialer Dialer, notification *notification, opts ...proxyOption) error {
-	if notification.ClientMode != Destination {
-		return errors.New("unsupported client mode")
+// ProxyDestination local connection via IoT secure tunneling.
```

Comment:
I think the document should mention about "destination device" concept. I guess this method is supposed to be called only on destination device and `ProxySource` is supposed to be called only on "source device". I think these differences are worth mentioning here.

Reply:
Updated the document